server/game/cards/14-HCWM/MarshalCavesCallarman.js

```diff
@@ -0,0 +1,35 @@
+const DudeCard = require('../../dudecard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class MarshalCavesCallarman extends DudeCard {
+    getBountyAmount(card) {
+        const bountyAmount = parseInt(card.bounty);
+
+        return bountyAmount;
+    }
+    setupCardAbilities(ability) {
+        this.persistentEffect({
+            condition: () => this.isWanted(),
+            match: this,
+            effect: [
+                ability.effects.dynamicBullets(() => -this.bounty),
+                ability.effects.dynamicInfluence(() => -this.bounty)
+            ]
+        });
+
+        this.action({
+            title: 'Reduce Bounty',
+            playType: 'noon',
+            condition: () => this.locationCard.hasKeyword('Public'),
```

Comment:
Should be `ifCondition` and add also `ifFailMessage`
```suggestion
            ifCondition: () => this.locationCard.hasKeyword('Public'),
            ifFailMessage: context => this.game.addMessage(),
```